code/modules/mob/mob_defines.dm

```diff
@@ -42,6 +42,7 @@
 	var/ear_damage = 0		//Carbon
 	var/stuttering = 0		//Carbon
 	var/slurring = 0		//Carbon
+	var/cultslurring = 0	//Carbon
```

Comment:
It would be a bit janky, but have negative values be cultslur, and positive be normal slur.

This would have the side effect of drunk people slurring LESS when culted though.